tasties_app/migrations/0003_create_categories.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations, transaction
+from tasties_app.models import Category
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tasties_app', '0001_initial'),
```

Comment:
Here, change `0001_initial` to `0002_alter_category_category_name`. This will make sure that `0002_alter_category_category_name` actually runs, and runs before this migration. If you get any trouble when migrating like that when testing on your local, just:
- `vagrant halt`
- throw away `db.sqlite3`
- `vagrant up`
- `vagrant provision`